dashboard-frontend/src/pages/NewStudyPage/form/configs/general.js

```diff
@@ -0,0 +1,113 @@
+export const general = {
+  type: 'configObject',
+  title: 'General',
+  description: 'The "general" configuration consists of... General stuff?',
```

Comment:
maybe we update this description to point o a document that has indepth explanation of what exactly this is (could just be our github for now)